rotkehlchen/chain/ethereum/manager.py

```diff
@@ -1195,3 +1195,23 @@ def get_basic_contract_info(self, address: ChecksumEthAddress) -> Dict[str, Any]
 
         self.contract_info_cache[address] = info
         return info
+
+    def _get_basic_contract_info_fallback(
```

Comment:
is not really a fallback since you are executing logic to process the data... what aobut `_process_contract_info`?